app/grandchallenge/algorithms/serializers.py

```diff
@@ -28,6 +28,7 @@ class Meta:
             "latest_ready_image",
             "pk",
             "title",
+            "result_template",
```

Comment:
Do we need to serialize this? I thought that we rely on Django to interpolate it?